Website/src/rendering/src/components/Checkout/LineItemCard.tsx

```diff
@@ -109,31 +109,33 @@ const LineItemCard = (props: LineItemCardProps): JSX.Element => {
     </button>
   );
 
-  const quantityBlock = props.editable ? (
+  const staticQuantityBlock = !props.editable && (
+    <p className="quantity-static">
+      <span className="quantity-label">Quantity: </span>
+      <span className="quantity-num">{props.lineItem.Quantity}</span>
+    </p>
+  );
+
+  const editableQuantityBlock = props.editable && (
     <>
       {quantityInput}
       {btnRemove}
       {btnWishList}
       {btnSaveLater}
     </>
-  ) : (
-    <p className="quantity-static">
-      <span className="quantity-label">Quantity: </span>
-      <span className="quantity-num">{props.lineItem.Quantity}</span>
-    </p>
   );
 
   const giftCheckbox = props.editable && (
     <GiftCheckboxLineItem lineItem={props.lineItem}></GiftCheckboxLineItem>
   );
 
   // TODO: add functionality to input
-  const userComment = (
+  const userComment = props.editable && (
```

Comment:
Even if not in the designs, could we please show the comments that the buyer added to each line item. Non-editable of course.